src/kvstore/raftex/RaftexService.cpp

```diff
@@ -94,11 +125,15 @@ void RaftexService::stop() {
         LOG(INFO) << "All partitions have stopped";
     }
     server_->stop();
+
+    // reclaim resource
+    waitUntilStop();
```

Comment:
Why move waitUntilStop inside stop.  
Typically, when we want to stop the service.  What we do is stop and waitUntilStop.  In which way, you could do some other  things between stop and waitUntilStop

Reply:
Because I see that we just use raft service in nebula store, and stop it in nebula store destructor, I want make it more simply, then move waitUntilStop in stop.

Reply:
Not only in NebulaStore.  Raft UTs also use it.

Reply:
fixed